modules/assignment/src/main/resources/assignment.properties

```diff
@@ -27,6 +27,14 @@ http.proxy.host:${http.proxy.host}
 http.proxy.port:${http.proxy.port}
 ruleCache.executor.pool.size:5
 
+#Specify the number of messages to be drained on the event of queue full (executor.max.queue.size).
+#Default value is 10_000
+drain.queue.by=10000
```

Comment:
These must be the default values just in case we need to implement queue draining functionality in wasabi open source.